emmo/.htaccess

```diff
@@ -0,0 +1,99 @@
+Options +FollowSymLinks
+
+# Rewrite engine setup
+RewriteEngine On
+
+
+# Rule 1: https://w3id.org/emmo
+#
+# If the request appears coming from a browser, redirect to the html documentation
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/
```

Comment:
In general, I would replace this rules with content negotiation / ACCEPT header rules

HTML requested (default browser action):
```
RewriteCond %{HTTP_ACCEPT} text/html [OR]
RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml
```

TURTLE requested (Browser, Script, Protegé, etc):
```
RewriteCond %{HTTP_ACCEPT} text/turtle [OR]
RewriteCond %{HTTP_ACCEPT} text/\* [OR]
RewriteCond %{HTTP_ACCEPT} \*/turtle
```

this can also be extented to futher serializations, e. g. JSON-LD:
```
RewriteCond %{HTTP_ACCEPT} application/ld\+json
```

Reply:
Thanks. Implemented